app/livechat/imports/server/rest/sms.js

```diff
@@ -1,11 +1,26 @@
+import { HTTP } from 'meteor/http';
 import { Meteor } from 'meteor/meteor';
 import { Random } from 'meteor/random';
 
+import { FileUpload } from '../../../../file-upload';
 import { LivechatRooms, LivechatVisitors, LivechatDepartment } from '../../../../models';
 import { API } from '../../../../api/server';
 import { SMS } from '../../../../sms';
 import { Livechat } from '../../../server/lib/Livechat';
 
+const fileStore = FileUpload.getStore('Uploads');
+
+const getUploadFile = async (details, fileUrl) => {
+	try {
+		const response = HTTP.get(fileUrl, { npmRequestOptions: { encoding: null } });
+		const buffer = new Buffer(response.content);
+		const size = buffer.length;
+		return fileStore.insertSync({ ...details, size }, buffer);
+	} catch (err) {
+		throw new Error(err.message);
```

Comment:
after a second look, I didn't get why you're using `try/catch` and just re-throwing the exception, is there a particular reason?